src/packages/utils/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v0.1.0 (2020-10-31)
```

Comment:
```suggestion
## v0.1.0 [#2246](https://github.com/excalidraw/excalidraw/pull/2246)
```
Also, the date can be removed as pr link will give the exact time and date of release, and in the case due to some unavoidable reason pr merge is delayed, we need not worry about changing the date unless automated

Reply:
the pull request link should be put on the change, not on the version header right?

Reply:
Yea, we most certainly won't be releasing a new npm package every commit to master, and either way we should  put changelog info into the body of the section, not into a header.